Gruntfile.js

```diff
@@ -6,37 +6,47 @@
  * Licensed under the MIT license.
  */
 
-'use strict';
-
 module.exports = function(grunt) {
-
+  'use strict';
   // Project configuration.
   grunt.initConfig({
 
     // Configuration to be run (and then tested).
     ftp_push: {
       default: {
         options: {
-          username: "{{Username}}",
-          password: "{{Password}}",
+          authKey: "serverA",
           host: "sample.server.com",
           dest: "/html/test/",
           port: 21
         },
         files: [ // Enable Dynamic Expansion, Src matches are relative to this path, Actual Pattern(s) to match
-          {expand: true,cwd: 'test',src: ['**/*']}
+          {expand: true,cwd: 'test',src: ['**/*']},
+          {expand: true,cwd: 'tasks',src: ['**/*'], dest: 'tasks/' }
+        ]
+      },
+
+      sample: {
```

Comment:
Not entirely sure we'd need a whole `sample` area here just to show the new option, to be honest. Can you please add the `debug` and new `files[]` options as items in the `README.md` instead?